### ISSUE TITLE
[API Question] Card Settings

### ISSUE THREAD

**Issue by rptl** [wekan/wekan#3062]
Is there any API call to perform "Card Settings" tasks?

**Comment by xet7** [comment]
@rptl 

No. You can add it to files at `wekan/models/*.js` as pull request.

**Comment by rptl** [comment]
@xet7 thanks.

One more Q. I'm assuming that Label and Board title changing has no webhook? Is that correct?

**Comment by xet7** [comment]
Please check from that code and docs at https://wekan.fi/api/
